akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -146,6 +146,8 @@ private[remote] class Remoting(_system: ExtendedActorSystem, _provider: RemoteAc
 
   import provider.remoteSettings._
 
+  private implicit val ec = system.dispatchers.lookup(Dispatcher)
```

Comment:
at least IntelliJ prefers to have explicit type on the implicits, `: ExecutionContext`

Reply:
I can see how that makes sense in general, but it'd be a lot of noise for the ec?